ironic/templates/configmaps/nova.conf.yaml.j2

```diff
@@ -0,0 +1,108 @@
+apiVersion: v1
+kind: ConfigMap
+data:
+  nova.conf: |+
+    [DEFAULT]
+    debug = {{ misc.debug }}
+    # TODO(DTadrzak): FIX ME
```

Comment:
@ss7pro I believe that we don't want to create dozens of conditionals. Common nova.conf will be used by nova-compute, ironic-compute, nova-api, nova-conductor, nova-scheduler, ... If we want to hide some config options for specific services, we'll end up with multiple dirty if-elif-else statements